[2024-05-26 22:36:40,238] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_new_data.transform 2024-05-26T22:33:41.250863+00:00 [queued]>
[2024-05-26 22:36:40,249] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_new_data.transform 2024-05-26T22:33:41.250863+00:00 [queued]>
[2024-05-26 22:36:40,249] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-26 22:36:40,249] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-05-26 22:36:40,249] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-26 22:36:40,255] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): transform> on 2024-05-26T22:33:41.250863+00:00
[2024-05-26 22:36:40,259] {standard_task_runner.py:52} INFO - Started process 287 to run task
[2024-05-26 22:36:40,263] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_new_data', 'transform', '2024-05-26T22:33:41.250863+00:00', '--job-id', '459', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_new_data.py', '--cfg-path', '/tmp/tmp1h9b6uno', '--error-file', '/tmp/tmp4yk4mezx']
[2024-05-26 22:36:40,270] {standard_task_runner.py:77} INFO - Job 459: Subtask transform
[2024-05-26 22:36:40,374] {logging_mixin.py:104} INFO - Running <TaskInstance: get_new_data.transform 2024-05-26T22:33:41.250863+00:00 [running]> on host 1f334d963743
[2024-05-26 22:36:40,414] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_new_data
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2024-05-26T22:33:41.250863+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-26T22:33:41.250863+00:00
[2024-05-26 22:36:41,342] {python.py:118} INFO - Done. Returned value was: None
[2024-05-26 22:36:41,349] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=get_new_data, task_id=transform, execution_date=20240526T223341, start_date=20240526T223640, end_date=20240526T223641
[2024-05-26 22:36:41,368] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-26 22:36:41,395] {local_task_job.py:146} INFO - Task exited with return code 0
